src/index.js

```diff
@@ -1 +1,3 @@
+import 'core-js/stable'
+import 'regenerator-runtime/runtime'
```

Comment:
This is based off the suggestion of babel docs -> https://babeljs.io/docs/en/babel-polyfill